src/WebJobs.Script.WebHost/Security/DefaultKeyValueConverterFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public sealed class DefaultKeyValueConverterFactory : IKeyValueConverterFactory
+    {
+        private static readonly PlaintextKeyValueConverter PlainTextConverter = new PlaintextKeyValueConverter();
+
+        public IKeyValueConverter GetValueConverter(Key key, KeyConversionAction action)
```

Comment:
You're right that this is not fully used at the moment and we'll be leveraging it once we introduce the encryption changes. I just kept it here for now because it is part of the factory interface.
Serializers do have a read/write method, but the idea here is that consumers could specify intent to give the factory more context about how the  intended use. They key scenario is where, regardless of the key state (encrypted state) the factory may decide to always use a crypto enabled serializer if writing. 
I guess another (cleaner) approach would be to have distinct interfaces for readers and writers. 
Thoughts?